konveyor/src/main/java/com/avito/konveyor/validation/ValidationPolicy.kt

```diff
@@ -1,6 +1,15 @@
 package com.avito.konveyor.validation
 
+import com.avito.konveyor.ItemBinder
+import com.avito.konveyor.konveyorConfiguration
 
+/**
+ * If set to `true` it will make [ItemBinder] throw in case of collisions or unregistered data
```

Comment:
No javadoc for ItemBinder? Even if we make it internal (?) it'd help to understand the abstractions. There is no info about ItemBinder in README either, though you're using it in code snippet